tox.ini

```diff
@@ -34,5 +34,7 @@ skip_install = true
 [testenv:mypy]
 basepython = python3.8
 deps = mypy
-commands = mypy {posargs} src
+extras = test
+    trio
+commands = mypy {posargs} .
```

Comment:
Should this be `mypy {posargs} src tests`, so as not to check everything in the project dir?

Reply:
Everything in the project dir type checks correctly, add ignores as new files get added

Reply:
There's a reason I've specified `src tests` for the other linters – virtualenvs, debugging scripts and whatnot – things that don't belong in VCS but are typically found in local checkouts of projects.